schema/artist/index.js

```diff
@@ -139,6 +140,37 @@ export const ArtistType = new GraphQLObjectType({
             })
           ),
       },
+      suggested_artists: {
+        // '/me/suggested/artists'
```

Comment:
Eh, I would remove this comment. If you do feel that you should document this alongside the field/schema, you can use `description: "...."` (and that way it'll come up in auto-completes as well).